cm/base/BaseCM/cm_output.py

```diff
@@ -74,19 +74,17 @@ class CMOutput(Schema):
 def validate(output: Dict) -> Dict:
     """Validate the output of the CM based on the CM output schema."""
     output_schema = CMOutput()
-    out = output_schema.load(data=output)
-    return out
-
-
-# get the api url
-API_URL = os.environ.get("API_URL")
+    # validates and deserializes an input dictionary to an application-level
+    # data structure
+    return output_schema.load(data=output)
 
 
 def output_raster(raster_name, raster_fd):
     """Add a raster to the api."""
     files = {"file": (raster_name, raster_fd, "image/tiff")}
     try:
-        resp = requests.post(API_URL + "api/geofile/", files=files)
+        API_URL = os.environ.get("API_URL")
```

Comment:
Why is this in the function and not a global data from the application ?

Reply:
I don't know, I have only corrected a "local bug" for now.....

Reply:
I don't know, I have only corrected a bug "locally" for now...